src/Main.css

```diff
@@ -7,6 +7,12 @@ body {
   overflow: hidden;
 }
 
+#app,
+.ant-layout {
```

Comment:
i am against having antd layout things in the base-client, please move these to the project